requirements.txt

```diff
@@ -1,7 +1,4 @@
 fedmsg
 moksha.hub
 psutil
-PyOpenSSL
-stomper
```

Comment:
I believe you'll need to keep this dependency for the connection to the UMB to work.

Reply:
`stomp*` or similar is not imported anywhere in the code base; an image built when this dependency was removed was able to listen to a message bus using the STOMP protocol and all the unit tests passed. Am I missing something? :)

Reply:
Oh alright. The fedmsg RPM doesn't seem to require it which is why I'm surprised it works.